tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -154,27 +173,71 @@ def test_brick_evict_with_node_down(self):
                 brick_id = node.get('id')
                 break
 
-        # Bring down the glusterfs node
-        vm_name = node_ops.find_vm_name_by_ip_or_hostname(hostname)
-        self.addCleanup(
-            self._wait_for_gluster_pod_after_node_reboot, hostname)
-        self.addCleanup(node_ops.power_on_vm_by_name, vm_name)
-        node_ops.power_off_vm_by_name(vm_name)
-
-        # Wait glusterfs node to become NotReady
-        custom = r'":.status.conditions[?(@.type==\"Ready\")]".status'
-        for w in waiter.Waiter(300, 20):
-            status = openshift_ops.oc_get_custom_resource(
-                self.ocp_client, 'node', custom, hostname)
-            if status[0] in ['False', 'Unknown']:
-                break
-        if w.expired:
-            raise exceptions.ExecutionError(
-                "Failed to bring down node {}".format(hostname))
+        self._power_off_node_and_wait_node_to_be_not_ready(hostname)
 
         # Perform brick evict operation
         try:
             heketi_ops.heketi_brick_evict(h_node, h_server, brick_id)
         except AssertionError as e:
             if ('No Replacement was found' not in six.text_type(e)):
                 raise
+
+    @pytest.mark.tier4
+    def test_brick_evict_with_node_down(self):
```

Comment:
test_brick_evict_on_four_node

Reply:
This can be generic. So keeping it as it is.

Reply:
test_brick_evict_on_more_than_three_node_with_one_down

Reply:
Done